frontend/packages/db/supabase/migrations/20250425123357_add_rls_to_github_repositories.sql

```diff
@@ -0,0 +1,36 @@
+BEGIN;
+
+ALTER TABLE "public"."github_repositories" ENABLE ROW LEVEL SECURITY;
+
+CREATE POLICY "authenticated_users_can_select_org_github_repositories" 
+  ON "public"."github_repositories" 
+  FOR SELECT TO "authenticated" 
+  USING (("organization_id" IN ( 
+    SELECT "organization_members"."organization_id"
+    FROM "public"."organization_members"
+    WHERE ("organization_members"."user_id" = "auth"."uid"())
+  )));
+
+COMMENT ON POLICY "authenticated_users_can_select_org_github_repositories" 
+  ON "public"."github_repositories" 
+  IS 'Authenticated users can only view repositories belonging to organizations they are members of';
+
+CREATE POLICY "authenticated_users_can_insert_org_github_repositories" 
+  ON "public"."github_repositories" 
+  FOR INSERT TO "authenticated" 
+  WITH CHECK (("organization_id" IN ( 
+    SELECT "organization_members"."organization_id"
+    FROM "public"."organization_members"
+    WHERE ("organization_members"."user_id" = "auth"."uid"())
+  )));
```

Comment:
[aside]
this is for frontend/apps/app/features/projects/actions/addProject.ts . 
for future, we should remove this policy for security reason.

Reply:
[aside]

https://github.com/liam-hq/liam/blob/cf55e1f3addb83945266b9501792cfd58efc2ade/frontend/apps/app/features/projects/actions/addProject.ts#L8

Reply:
I'd like to revisit this in the future.
Should we leave a TODO comment to track it? @NoritakaIkeda